Sources/Queries/ObservableQueryRef.swift

```diff
@@ -213,12 +154,16 @@ public class QueryRefObservableObject<
 /// - ``data``: Published variable that contains bindable results of the query.
 /// - ``lastError``: Published variable that contains ``DataConnectError``  if last fetch had error.
 ///            If last fetch was successful, this variable is cleared
-@available(macOS 14, iOS 17, tvOS 17, watchOS 10, *)
+@available(macOS 15, iOS 17, tvOS 17, watchOS 10, *)
```

Comment:
Why the macOS version bump? The Observable macro looks to be available on macOS 14?

https://developer.apple.com/documentation/observation/observable